app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ApplicationRecord
+  validates :rating, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 1, less_than_or_equal_to: 10 }
+  validates :body, presence: true, length: { minimum: 10 }
+  
+  belongs_to :user
+  belongs_to :cape
+end
```

Comment:
Review model tests!

Reply:
Model tests written for review, cape, and user :)